packages/aws-serverless/scripts/buildLambdaLayer.ts

```diff
@@ -174,6 +174,8 @@ function buildPackageJson(): void {
     }
   }
 
+  resolutions['import-in-the-middle'] = '1.14.2';
```

Comment:
can we add a comment here for future reference, why we do this, and when we can remove this again?